### DISCUSSION TITLE
Why the need for ! in error messages?

### DISCUSSION THREAD

**Discussion by Peter101Pan** [duckdb/duckdb#11722] [General]
Hi 

Why the need for **!** in error messages?

As per 

Catalog Error: Scalar Function with name to_char does not exist **!**

Thanks

**Comment by Tishj** [comment]
No reason, is there any reason you are against the use of exclamation marks in error messages?

**Reply by Peter101Pan** [reply]
It's a bit strange, to see this type of feedback. I have not seen additional emphasis to error messages in other DB applications, Oracle/Postgres etc. given via the use of a !.  We already know there is an error/panic in play.  Why the need for exclamation, its almost offensive?
